src/components/custom/NewDocumentCard.js

```diff
@@ -72,7 +74,10 @@ export function NewDocumentCard({project}) {
             </AlertDialogDescription>
           </AlertDialogHeader>
           <AlertDialogFooter>
-            <AlertDialogAction onClick={handleDialogClose}>
+            <AlertDialogAction onClick={() => {
+              setIsDialogOpen(false);
+              router.push(`/projects/${encodeURI(project.pid)}/document/${encodeURI(documentId)}`);
+            }}>
```

Comment:
nit: this should all be in the handleDialogClose method. That method is currently not used, but was previously.

Reply:
code left as is. unused method deleted